### PR TITLE
Init

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/COM-1067

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
 ```
 
-Run the test suite using this command:
+Create `.env` file with following contents
+```env
+EXASOL_HOST=
+EXASOL_PORT=
+EXASOL_USERNAME=
+EXASOL_PASSWORD=
+EXASOL_SCHEMA=
```
Comment (michaljurecko) [inline README.md:23]: Tu by som doplnil, ... 
```
To use a local instance of Exasol via docker-compose, enter:
EXASOL_HOST=exasol
EXASOL_PORT=8563
EXASOL_USERNAME=sys
EXASOL_PASSWORD=exasol
EXASOL_SCHEMA=testSchema
```

File: src/ConnectionFactory.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Keboola\TableBackendUtils\Escaping\Exasol\ExasolQuote;
+use Psr\Log\LoggerInterface;
+
+class ConnectionFactory
+{
+    public static function createFromConfig(Config $config, LoggerInterface $logger): Connection
+    {
+        $logger->info(sprintf('Connection to "%s".', $config->getDatabaseHost()));
+        $connection = ExasolConnection::getConnection(
+            sprintf('%s:%s', $config->getDatabaseHost(), $config->getDatabasePort()),
```
Comment (michaljurecko) [inline src/ConnectionFactory.php]: Schema sa da dat priamo do connection string, ... ak by sa robil napr. reconnect v pripade vypadku spojenia, tak je to vyhoda, ze sa nemusi volat dalsi prikaz na nastavenie schemy:
https://github.com/keboola/db-writer-exasol/blob/9d12265236fb815447c7a080cacb182a955c69df/src/ConnectionFactory.php#L23

File: src/ConnectionFactory.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        $connection->connect();
+
+        $logger->info(sprintf('Use schema "%s".', $config->getDatabaseSchema()));
+        $connection->executeStatement(sprintf(
+            'OPEN SCHEMA %s',
+            ExasolQuote::quoteSingleIdentifier($config->getDatabaseSchema())
+        ));
+
+        $connection->executeQuery(sprintf(
+            'ALTER SESSION SET QUERY_TIMEOUT=%d',
```
Comment (michaljurecko) [inline src/ConnectionFactory.php]: Query timeout sa da tiez dat do connection string:
https://docs.exasol.com/connect_exasol/drivers/odbc/using_odbc.htm

File: tests/functional/manifest-ok/expected/data/out/tables/table1.manifest [hunk 4]
```diff
@@ -0,0 +1 @@
+{"columns":["product_name","price","count"],"column_metadata":{"product_name":[{"key":"KBC.datatype.type","value":"VARCHAR"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"100"}],"price":[{"key":"KBC.datatype.type","value":"DOUBLE PRECISION"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"FLOAT"},{"key":"KBC.datatype.length","value":"64"}],"count":[{"key":"KBC.datatype.type","value":"DECIMAL"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"NUMERIC"},{"key":"KBC.datatype.length","value":"36,0"}]}}
```
Comment (michaljurecko) [inline tests/functional/manifest-ok/expected/data/out/tables/table1.manifest]: Manifest by sa mohol v testoch spravit `pretty printed`, aby sa dalo v dalsich PR nejak rozumne zistit, co sa zmenilo:
https://github.com/keboola/db-extractor-mysql/blob/2bcf2c1d1e5d129ffa9e69aca26389ad2f55541e/tests/functional/DatadirTest.php#L77

File: tests/traits/GetTableColumnsTrait.php [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+trait GetTableColumnsTrait
+{
+    public function getTableColumns(Connection $connection, array $table): array
+    {
+        $sqlTemplate = 'select * from EXA_ALL_COLUMNS WHERE "COLUMN_SCHEMA" = \'%s\' AND "COLUMN_TABLE" = \'%s\';';
+
+        return $connection->executeQuery(
+            sprintf(
+                $sqlTemplate,
+                $table['TABLE_SCHEMA'],
```
Comment (michaljurecko) [inline tests/traits/GetTableColumnsTrait.php]: `ExasolQuote`

File: tests/traits/GetTablesTrait.php [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+use Doctrine\DBAL\Connection;
+
+trait GetTablesTrait
+{
+    public function getTables(Connection $connection): array
+    {
+        $sqlTemplate = 'select * from EXA_ALL_TABLES WHERE "TABLE_SCHEMA" = \'%s\';';
+
+        return $connection->executeQuery(
+            sprintf($sqlTemplate, (string) getenv('EXASOL_SCHEMA'))
```
Comment (michaljurecko) [inline tests/traits/GetTablesTrait.php]: `ExasolQuote`

**Review by michaljurecko — APPROVED** [review]
LGTM, par drobnosti na zvazenie

**Comment by ondrajodas** [comment]
@webrouse díky, připomínky jsem zapracoval...

**Pull request merged**